internal/pkg/api/server/templates/service/service.go

```diff
@@ -25,8 +25,11 @@ import (
 type service struct{}
 
 func New(d dependencies.Container) (Service, error) {
-	if err := StartPullCron(d); err != nil {
-		return nil, err
+	defaultRepo := d.DefaultRepository()
+	if defaultRepo.Type == model.RepositoryTypeGit {
+		if err := StartPullCron(d); err != nil {
+			return nil, err
+		}
```

Comment:
Sorry, ale este to IMHO nie je dobre :slightly_smiling_face: 
Pazdy projekt moze mat (teoreticky) vlastne repository, takze to ci je default repository `type=dir` neovplyvnuje to, ci sa ma spustit cron.

`RepositoryManager` ma pracovat iba s `git repository`.

Preto som na calle navrhoval pridat `AddRepository` aj s podmienkou sem:
https://github.com/keboola/keboola-as-code/blob/fe89c7f1d624da86baca9b005b9fdd14a0f7e74a/internal/pkg/api/server/templates/dependencies/dependencies.go#L148

A vyhodit `v.defaultRepository` parameter z `NewManager` tu:
https://github.com/keboola/keboola-as-code/blob/fe89c7f1d624da86baca9b005b9fdd14a0f7e74a/internal/pkg/api/server/templates/dependencies/dependencies.go#L144

Reply:
+ idealne este pridat podmienku do `AddRepository`, `type != git` - > `panic`.

Reply:
omg, `AddRepository` do `dependecnies.RepositoryManager()`! Uuž snad vím. Jsem pořád nemohl pochopit jak přestat předávat `defaultRepository` v `NewManager`. 

Ten panic už tam je (pokud je teda na správným místě 😅): https://github.com/keboola/keboola-as-code/blob/3b4252ac0ee5c3889e2a4bf3b69ccc949a5ff18f/internal/pkg/template/repository/manager.go#L43-L46

A jo, to je pravda s tím cronem, ten nemůžu jen tak vypnout. Fixnu. 🙂